### ISSUE TITLE
Fix overlapping menus/cards

### ISSUE THREAD

**Issue by timothyfcook** [saxifrage/caac-map#33]
I'm not sure what the best UI solution is here. Two options:
1. The menus are folded up by default, only fold down when you hover over them
2. The menus are folded down, but are smaller and are always in the top-left, while the focused map and selected card are dead center so they don't overlap.

Option 1 is probably the easiest to implement.

![screen shot 2015-08-03 at 3 00 51 pm](https://cloud.githubusercontent.com/assets/1823300/9046335/fd9ea17c-39f7-11e5-8c8f-a21bb4a6dc9e.png)

**Comment by chelseaerdner** [comment]
I think option 1 and having the menus closed would be the best approach. Personally, I think we don't want that much focus to be on the menus and with them exposed, they take up a lot of space.

**Comment by cameronscott137** [comment]
My vote is also for option 1. However, it probably makes sense to keep the menu that is in focus exposed, eg. if I'm looking at the "DIY Gardner" resource, "Food & Gardening" and "Gardening" can shrink up, I probably want to see related DIY Gardner resources by default.

To compensate for the exposed nav, we can shrink the overall point size, and I can adjust the z-index so that the nav always appears overtop any exposed popover:

![screen shot 2015-08-03 at 4 15 07 pm](https://cloud.githubusercontent.com/assets/2473472/9046753/ee92654e-39fa-11e5-9241-099ecf4b89d0.png)

**Comment by cameronscott137** [comment]
Playing around with this more, I think shrinking up the nav and running option 1 is definitely the way to go

![screen shot 2015-08-03 at 5 57 44 pm](https://cloud.githubusercontent.com/assets/2473472/9048722/54279312-3a09-11e5-8a3b-60fb92b3ef44.png)

**Comment by chelseaerdner** [comment]
Yeah, shrinking it looks cleaner and like I mentioned to you yesterday..the nav wasn't supposed to be the focus. We want users to navigate the map via the zoom/scrolling of the canvas. I think with the nav larger, it is too prominent and over powers all the work that went into the map itself.

**Comment by timothyfcook** [comment]
Exactly. 

Let's also reduce the size of the nav-blocks and font-size.

We can make box-width closer to 200px, and reduce the vertical padding to make the overall footprint smaller.

**Comment by chadwhitacre** [comment]
[!m](http://motivate.im/) @chelseaerdner @cameronscott137 

I [turned off the nav](https://github.com/saxifrage/cityasacampus/commit/de6075877a5efed9569ae0af73eeaa5526875eea#diff-7274b05633c2b7471d969b11ddb5f95aR202) because it was so distracting while trying to work on the map itself. I forgot to turn it back on! :scream_cat:
